doc/nrf/ug_thread_ot_integration.rst

```diff
@@ -0,0 +1,130 @@
+.. _openthread_integration:
+
+OpenThread Integration
+######################
+
+.. contents::
+    :local:
+    :depth: 2
+
+The goal of this page is to explain how the OpenThread stack is integrated with remaining parts of the system in terms of the file structure,
+used threads and synchronization primitives and the network traffic flow.
```

Comment:
```suggestion
This page explains how the OpenThread stack is integrated with Zephyr and the nRF Connect SDK. The location of components in the ffile structure, used threads and synchronization primitives, and the network traffic flow are described.
```

Reply:
Will commit this change manually (`ffile` ;) )